package.json

```diff
@@ -11,13 +14,12 @@
     "uid2": "0.0.3"
   },
   "devDependencies": {
-    "co": "^4.5.4",
-    "co-mocha": "^1.1.2",
+    "ava": "^0.21.0",
     "istanbul": "^0.4.2",
     "koa": "^1.1.2",
     "koa-json-body": "^4.0.0",
     "koa-route": "^2.4.2",
-    "mocha": "^2.3.4",
-    "standard": "^7.1.2"
+    "sinon": "^2.4.1",
+    "standard": "^10.0.2"
```

Comment:
Had to update `standard`, because it was crashing when encountering `async/await`.